### PR TITLE
Hotfix/0.1.1

### PR DESCRIPTION
**Pull request by byb121**
* gtftools.py will not create a file along side input gtf file when converting from GENCODE to ENSEMBL format
* get_tpm_fpkm.py can optionally use an output folder.
* merge_samples.py tells how many lines to ignore on the fly.

### CODE REVIEW

**Comment by byb121** [comment]
Do not merge. I'll merge it at my end when you think it looks ok.

**Review by sb43 — APPROVED** [review]
As long as it passes all the tests ( which is I guess comparison with output files), I am fine with the changes. As discussed please email to author for gtftools script.

**Pull request closed without merging**
